zcash_client_backend/CHANGELOG.md

```diff
@@ -79,6 +82,10 @@ and this library adheres to Rust's notion of
 
 ### Removed
 - `zcash_client_backend::data_api`:
+  - `AccountBalance::unshielded`. `AccountBalance` no longer provide the `unshielded` value returning a `NonNegativeAmount` for all transparent funds. Use`unshieded_balance` instead which provides `Balance` value.
+  - `AccountBalance::add_unshielded_value` was removed with the `unshielded` value as
+  described above. Use the appropriate `add_*_value` method(s) of `Balance` on its argument
+  instead.
```

Comment:
These two lines should be indented another two spaces. (Markdown is picky about indentation.)